synapseclient/client.py

```diff
@@ -178,6 +178,27 @@ def _test_import_sftp():
              "\n\n\n"))
         raise
 
+def _extract_zip_file_to_cache(zip_file, zip_entry_name, cache_dir):
+    """
```

Comment:
Make more generic: extract files into specified directory.  Move to synapseclient/utils.py